src/plugins/intel_cpu/src/nodes/tensoriterator.cpp

```diff
@@ -645,7 +645,7 @@ void TensorIterator::execute(const dnnl::stream& strm) {
     int max_num_iter = trip_count_check->getStatus();
 
     for (auto& mapper : first_mappers) {
-        mapper.second->execute(strm);
+        mapper.second->execute(strm, -1);
```

Comment:
This interface also seems to be a bit inconsistent in term of using this optional value, but fixing it requires too many changes, so let's stick with this change for now.

Reply:
Agree, so I guess no action required here for now. Right?

Reply:
That's right.